docs/configuring-playbook-bridge-appservice-irc.md

```diff
@@ -0,0 +1,435 @@
+# Setting up Appservice IRC (optional)
+
+The playbook can install and configure [matrix-appservice-irc](https://github.com/TeDomum/matrix-appservice-irc) for you.
+
+See the project's [documentation](https://github.com/TeDomum/matrix-appservice-irc/blob/master/HOWTO.md) to learn what it does and why it might be useful to you.
+
+You'll need to use the following playbook configuration:
+
+```yaml
+matrix_matrix_appservice_irc_enabled: true
+matrix_appservice_irc_configuration_extension_yaml: |
```

Comment:
Maybe posting the full contents of `matrix_appservice_irc_configuration_extension_yaml` makes it overly long (and hard to keep in sync with the other example of it that we have in `roles/matrix-synapse/defaults/main.yml`).

Maybe it would be better if we have a minimal example and point people to the ``roles/matrix-synapse/defaults/main.yml`` instead?

That way, they can see the default configuration in `matrix_appservice_irc_configuration_yaml` too.